frontend/src/components/worksheets/WorksheetNameSearch.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import queryString from 'query-string';
+
+export default class extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            loading: true,
+            error: false
+        }
+    }
+    async componentDidMount() {
+        const { name } = queryString.parse(this.props.location.search);
+        try {
+            const response = await fetch(`/rest/worksheets?specs=${name}`).then(e => e.json());
```

Comment:
why `specs` instead of `name`, as it was before?

Reply:
See this:
https://github.com/codalab/codalab-worksheets/blob/cfb1d8d5ffef5b03c192c75a4cc9a7796867d693/codalab/rest/worksheets.py#L119
Specifying `specs` will return just one result, and also create a worksheet if it hasn't already been created -- this is the same behavior as that of the `/rest/worksheets/?name=dashboard` endpoint.